src/suspendplugin.c

```diff
@@ -237,11 +248,11 @@ suspend_handle_display_on_off_iface(
     DBG("iface: %s suspend: %d\n", ifname, (int)on_off ? 0 : 1);
 
     msg = nlmsg_alloc();
-    cb = nl_cb_alloc(NL_CB_DEFAULT);
 
     genlmsg_put(msg, 0, 0, driver_id, 0, 0, NL80211_CMD_TESTMODE, 0);
 
     struct testmode_cmd_suspend susp_cmd;
+    memset(&susp_cmd, 0, sizeof(struct testmode_cmd_suspend));
```

Comment:
personal preference, feel free to ignore ... avoiding sizeof(type) => no need to check if that type is correct

    memset(&susp_cmd, 0, sizeof susp_cmd);

Reply:
ok :)